integration_tests/snaps/stage-package-version/snap/snapcraft.yaml

```diff
@@ -0,0 +1,13 @@
+name: stage-package-version
+version: '0.1'
+summary: install a specific, but non-existent version of a stage-package
+description: |
+  Install a specifi, but non-existent  version of  a stage-package
```

Comment:
I thought that as well but the apt-cache bits are mocked in the unit tests.  Unless I'm missing something.  It seemed like I'd be testing something so faked that it couldn't fail unless I mis-wrote the test.  I'm open to ideas though.